.github/workflows/release-backend-app-generic.yml

```diff
@@ -127,21 +127,43 @@ jobs:
       # TODO deduplicate this code when github actions allows to easily share a script used
       # by reusable workflows. Currently only the workflow file is checked out at the caller
       # There are no good workarounds to checkout a companion file (script or composite action)
-      - name: Check if gitReference is already released
+      - name: Check if gitReference is missing already released commits
         run: |
-          # needs fetch-depth: 0
-          # This ensures that an old commit (potentially with a security issue) can not be rereleased
-          # as a new version (thus potentially restoring a security issue). This will not block regular
-          # releases because there is always at least the version bump commit in a real new release even if
-          # we are just rereleasing the same code with a new name as part of a global release.
-          if ! release_tags=$(git tag --contains "$RUNGHA_GIT_REFERENCE" 'v[0-9]*'); then
-            echo "ERROR checking whether gitRef $RUNGHA_GIT_REFERENCE is already released or not."
-            exit 1
-          elif [ -n "$release_tags" ]; then
-            echo "ERROR: gitRef $RUNGHA_GIT_REFERENCE already released: $release_tags"
-            exit 1
+          # This ensures that we don't silently drop commits (such as security fixes) by
```

Comment:
This message is not clear. No commits may be removed from the history, but some may be "ignored" by the new release.
I suggest to use something like:
```
The following block ensures that the new release will not be made from a commit
that is an ancestor of the last release of the branch.
If this case was allowed, then it would be possible to release a version
with less commits than the last released version (and, for instance, without some potential security fixes).
If the code that finds existing releases fails and doesn't find anything, it
could be the first release, or it could be that there is a bug or exploit. In
this case, instead of allowing to release anything, we just allow to release the
tip of main (allowing to release anything would also allow to omit commits).
```